### ISSUE TITLE
Version number of Cabal on 2.0 branch is incorrect

### ISSUE THREAD

**Issue by bgamari** [haskell/cabal#4611]
The version number listed in `Cabal.cabal` is 2.0.0.0 whereas the tag claims the release should be 2.0.0.1.

Thanks to @hvr for noticing this.

**Comment by bgamari** [comment]
Just to be clear, `ghc` 8.2.1 is blocked on this.

**Comment by 23Skidoo** [comment]
Fixed in 89ef359f8c0fa25e54ee79d4762472941098251e.

**Comment by hvr** [comment]
@23Skidoo ...release lib:Cabal-2.0.0.2 to Hackage? :-)

**Comment by thekyriarchy** [comment]
If this was fixed in July, how come a current download of Haskell platform off the haskell.org website results in a broken cabal that can't update?

**Comment by phadej** [comment]
@thekyriarchy This issue was about pre-release stuff.

Could you expand your problem, please open a new issue for it.

**Comment by thekyriarchy** [comment]
```
~> ghc --version
The Glorious Glasgow Haskell Compilation System, version 8.2.1
~> cabal --version
cabal-install version 2.0.0.0
compiled using version 2.0.0.2 of the Cabal library 
```

```
~> cabal info Cabal
* Cabal            (library)
    Synopsis:      A framework for packaging Haskell software
    Versions available: 1.20.0.3, 1.20.0.4, 1.22.6.0, 1.22.7.0, 1.22.8.0,
                        1.24.0.0, (1.24.1.0), 1.24.2.0, 2.0.0.2 (and 42 others)
    Versions installed: 2.0.0.2
    Homepage:      http://www.haskell.org/cabal/
    Bug reports:   https://github.com/haskell/cabal/issues
    Description:   The Haskell Common Architecture for Building Applications and
                   Libraries: a framework defining a common interface for
                   authors to more easily build their Haskell applications in a
                   portable way.
                   
                   The Haskell Cabal is part of a larger infrastructure for
                   distributing, organizing, and cataloging Haskell libraries
                   and tools.
    Category:      Distribution
    License:       BSD3
    Author:        Cabal Development Team <cabal-devel@haskell.org>
    Maintainer:    cabal-devel@haskell.org
    Source repo:   https://github.com/haskell/cabal/
    Flags:         bundled-binary-generic, old-directory, parsec,
                   parsec-struct-diff
    Dependencies:  array >=0.1 && <0.6, base >=4.5 && <5,
                   bytestring >=0.9 && <1, containers >=0.4 && <0.6,
                   deepseq >=1.3 && <1.5, filepath >=1.3 && <1.5,
                   pretty ==1.1.*, process >=1.1.0.1 && <1.7,
                   time >=1.4 && <1.9, directory ==1.1.*, old-time >=1 && <1.2,
                   process >=1.0.1.1 && <1.1.0.2, directory >=1.2 && <1.4,
                   process >=1.1.0.2 && <1.7, binary >=0.5 && <0.7,
                   binary >=0.7 && <0.9, ghc-prim ==0.2.*, unix >=2.5 && <2.8,
                   Win32 >=2.2 && <2.6, transformers -any,
                   parsec >=3.1.9 && <3.2, keys -any,
                   singleton-bool >=0.1.1.0 && <0.2, these >=0.7.1 && <0.8,
                   generics-sop ==0.2.*, Cabal -any, filepath -any,
                   directory -any, bytestring -any, tar ==0.5.*,
                   containers -any, base -any, Cabal -any,
                   tasty-quickcheck -any, tasty-hunit -any, tasty -any,
                   filepath -any, bytestring -any, base -any, Cabal -any,
                   QuickCheck >=2.7 && <2.11, pretty -any, tagged -any,
                   tasty-quickcheck -any, tasty-hunit -any, tasty -any,
                   filepath -any, directory -any, containers -any, base -any,
                   array -any
    Documentation: /usr/local/haskell/ghc-8.2.1-x86_64/share/doc/ghc-8.2.1/html/libraries/Cabal-2.0.0.2
    Cached:        No
    Modules:
        Distribution.Backpack
        Distribution.Backpack.ComponentsGraph
...
```


```
~> cabal install cabal-install
Resolving dependencies...
Configuring hackage-security-0.5.2.2...
Building hackage-security-0.5.2.2...
Failed to install hackage-security-0.5.2.2
Build log ( /home/kxra/.cabal/logs/ghc-8.2.1/hackage-security-0.5.2.2-AbKfMD8oU3gCavTAiZOKOv.log ):
cabal: Entering directory '/tmp/cabal-tmp-12089/hackage-security-0.5.2.2'
Configuring hackage-security-0.5.2.2...
Preprocessing library for hackage-security-0.5.2.2..
Building library for hackage-security-0.5.2.2..
<command line>: cannot satisfy -package-id parsec-3.1.11-DPgnR92AWEaFOaixmwipet: 
    parsec-3.1.11-DPgnR92AWEaFOaixmwipet is unusable due to shadowed dependencies:
      mtl-2.2.1-19EL8AGBsN3DnnOhrC9xY3 text-1.2.2.2-EGUst8sqNAZCw1xLPcmcMH
    (use -v for more information)
cabal: Leaving directory '/tmp/cabal-tmp-12089/hackage-security-0.5.2.2'
cabal: Error: some packages failed to install:
cabal-install-2.0.0.0-DAqBbhdtA06LrVF2rumJTX depends on cabal-install-2.0.0.0
which failed to install.
hackage-security-0.5.2.2-AbKfMD8oU3gCavTAiZOKOv failed during the building
phase. The exception was:
ExitFailure 1
```

**Comment by 23Skidoo** [comment]
@thekyriarchy This looks like #4749. Anyway, `cabal-install` 2.0.0.0 and Cabal 2.0.0.2 are the latest versions at the moment.
